polaris-tokens/src/token-groups/colors.ts

```diff
@@ -576,6 +576,75 @@ export const colors = {
   'surface-attention': {
     value: 'rgba(255, 234, 138, 1)',
   },
+  'surface-magic': {
+    value: 'rgba(233, 217, 252, 1)',
+    description: 'For use as background color of Shopify Magic components',
+  },
+  'surface-magic-subdued': {
+    value: 'rgba(246, 240, 253, 1)',
+    description:
+      'For use as subdued background color of Shopify Magic components',
+  },
+  'surface-magic-subdued-hoverd': {
+    value: 'rgba(238, 223, 255, 1)',
+    description: 'Hover state for magic subdued',
+  },
+  'surface-magic-subdued-pressed': {
+    value: 'rgba(230, 209, 255, 1)',
+    description: 'Pressed state for magic subdued',
+  },
+  'surface-magic-subdued-depressed': {
```

Comment:
For `surface-magic-subdued-depressed` the description should be `Depressed` state (it says `Pressed`.)
Also, on the other descriptions, it's referred to as `Shopify Magic` and on the surface colors it's called `magic`. For consistency, maybe update the surface color descriptions to be `Shopify Magic`?